SendUserDataProcessingEmailActivity/handler.ts

```diff
@@ -91,40 +67,75 @@ export const getSendUserDataProcessingEmailActivityHandler = (
           <title>${subject}</title>
         </head>
         <body>
-          <p>${emailText}</p>
+          <p>${emailText.replace("\n", "<br>\n")}</p>
         </body>
       </html>`;
-      // Send an email to the DPO containing the information about the IO user's
-      // choice to download or delete his own private data stored by the platform
-      const errorOrSentMessageInfo = await sendMail(mailerTransporter, {
-        from,
-        html: documentHtml,
+
+export const getDpoEmailSubject = (
+  choice: UserDataProcessingChoiceEnum,
+  fiscalCode: FiscalCode
+) => `IO - Richiesta di tipo ${choice} da parte di ${fiscalCode}`;
+
+const failActivity = (context: Context, logPrefix: string) => (
+  errorMessage: string,
+  errorDetails?: string
+) => {
+  const details = errorDetails ? `ERROR_DETAILS=${errorDetails}` : ``;
+  context.log.error(`${logPrefix}|${errorMessage}|${details}`);
+  return ActivityResultFailure.encode({
+    kind: "FAILURE",
+    reason: errorMessage
+  });
+};
+
+const success = () =>
+  ActivityResultSuccess.encode({
+    kind: "SUCCESS"
+  });
+
+/**
+ * For each user data procesing request send an email to the DPO
+ * containing the information about the user's choice
+ * to download or delete his own private data stored by the platform
+ */
+export const getSendUserDataProcessingEmailActivityHandler = (
+  emailDefaults: EmailDefaults,
+  sendMail: ReturnType<sendMailTaskT>,
+  findOneProfileByFiscalCode: ReturnType<findOneProfileByFiscalCodeTaskT>,
+  logPrefix = "SendUserDataProcessingEmail"
+) => async (context: Context, input: unknown) => {
+  const failure = failActivity(context, logPrefix);
+  return fromEither(ActivityInput.decode(input))
```

Comment:
when possible avoid assignement to `.value` and calls to `isLeft` / `isSome`
use fp-ts methods to transform an input into the expected output